packages/design-picker/src/components/theme-preview/index.tsx

```diff
@@ -79,8 +82,10 @@ const ThemePreview: React.FC< ThemePreviewProps > = ( {
 		};
 	}, [ setIsLoaded, setViewport ] );
 
+	// Ideally the iframe's document.body is already available on isLoaded = true.
+	// Unfortunately that's not always the case, so isFullyLoaded serves as another retry.
```

Comment:
Checking the embedded page JS file fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Swrgcnpx%2Qvsenzr%2Qrzorq%2Swrgcnpx%2Qvsenzr%2Qrzorq.wf%3Se%3Q1no2p2n9%23150-og 

`partially-loaded` is fired on script execution, which might be the reason why sometimes it might be run before the window `load` event, which waits for everything to be loaded.